### ISSUE TITLE
Add mosaic adjective to AD inheritance

### ISSUE THREAD

**Issue by wrightmw** [ClinGen/clincoded#1942]
Add "mosaic" as an adjective for Automosomal Dominant inheritance:

![Screen Shot 2019-06-11 at 8 17 02 AM](https://user-images.githubusercontent.com/15131169/59284659-78ccae00-8c21-11e9-865a-3521311aa7ce.png)

**Comment by wrightmw** [comment]
@erinriggs Would you also want this for the GCI?:

![Screen Shot 2019-06-11 at 8 17 48 AM](https://user-images.githubusercontent.com/15131169/59284726-9c8ff400-8c21-11e9-916c-df379cfaf1a2.png)

**Comment by wrightmw** [comment]
@marinadistefano
@jennygoldstein
@courtneythaxton

**Comment by ErinRiggs** [comment]
Should we hold off on this until GenCC comes up with the uniform terms they want to use to describe mode of inheritance @marinadistefano?  I know they were discussing how to describe the idea of "mosaic," and I'm guessing that won't be the term we ultimately use.

**Comment by marinadistefano** [comment]
This is so that the Brain Malformations group can start approving their variants and submitting to ClinVar, so I'm not sure if we can wait for GenCC.

________________________________

**Comment by ErinRiggs** [comment]
Ah, got it.

**Comment by courtneythaxton** [comment]
I do not think we would need mosaic for the GCI, I would assume that we would not curate an entire gene-disease relationship on mosaic individuals as that would not be Mendelian. However, there are some individuals in families that start off as mosaic and then the variant segregates as Mendelian in other generations, but we usually note this in the family Info section.

What do you think @marinadistefano @ErinRiggs

**Comment by marinadistefano** [comment]
I think the adjectives are the same across GCI and VCI and we need it for VCI, but yes, we probably wouldn't use it for gene curation. I'm not sure if we can make them differ though?

________________________________

**Comment by courtneythaxton** [comment]
Hopefully they can make it contingent on choosing new "variant curation" button on the homepage vs  new "gene curation." IS that possible @wrightmw

**Comment by wrightmw** [comment]
@courtneythaxton Yes, we can apply this to just the VCI, for now. You can think over what you want for the GCI and we can add this in (if required) at a later date.

**Comment by ErinRiggs** [comment]
I think that, since this sounds time sensitive, go ahead and add it to the VCI and leave it off of the GCI for now - if and when we want to add it there, we can.  Thanks!

**Comment by bryanwulf** [comment]
At this time, there's one source of MOI adjectives for both tools (GCI and VCI). Since multiple members of the GCI WG were "ok" with this new adjective appearing in the GCI, no additional effort to create tool-specific lists was made.
